src/simulator_core/entities/assets/production_cluster.py

```diff
@@ -267,7 +261,7 @@ def update(self) -> None:
             # Set the setpoints (Ts, Tr, Qh)
             self.set_setpoints(setpoints)
             # Set massflow
-            self._controlled_mass_flow = self.pandapipes_net.res_circ_pump_mass[
```

Comment:
Is this correct?

Reply:
I put the correct attributes back. I also correct the check simulation performed method